dts/bindings/spi/atmel,sam-spi.yaml

```diff
@@ -18,3 +18,16 @@ properties:
       type: int
       description: peripheral ID
       required: true
+
+    pinctrl-0:
+      type: phandles
+      required: true
+      description: |
+        PIO pin configuration for the various SPI signals that includes
+        MISO, MOSI, SPCK, and possibly various chipselects signals.  We
+        expect that the phandles will reference pinctrl nodes.  These
+        nodes will have a nodelabel that matches the Atmel SoC HAL defines
+        and be of the form p<port><pin><periph>_<inst>_<signal>.
+
+        For example the SPI on SAME7x would be
+           pinctrl-0 = <&pd20b_spi0_miso &pd21b_spi0_mosi &pd22b_spi0_spck>;
```

Comment:
Just curious, is below a valid configuration?
<&pd20b_spi0_miso &pd21b_spi0_mosi &pd22b_spi0_spck &pb2d_spi0_npcs0 &pa31a_spi0_npcs1 &pd12c_spi0_npcs2 &pd27b_spi0_npcs3 >;

Reply:
Yes it is.